src/CentrifugeRouter.sol

```diff
@@ -86,27 +80,37 @@ contract CentrifugeRouter is Auth, ICentrifugeRouter {
         require(owner == _initiator || owner == address(this), "CentrifugeRouter/invalid-owner");
 
         lockedRequests[controller][vault] += amount;
-        SafeTransferLib.safeTransferFrom(poolManager.getVaultAsset(vault), owner, address(escrow), amount);
+        (address asset,) = poolManager.getVaultAsset(vault);
+        SafeTransferLib.safeTransferFrom(asset, owner, address(escrow), amount);
+
         emit LockDepositRequest(vault, controller, owner, _initiator, amount);
     }
 
     /// @inheritdoc ICentrifugeRouter
     function openLockDepositRequest(address vault, uint256 amount) external payable protected {
         open(vault);
-        lockDepositRequest(vault, amount, _initiator, _initiator);
+
+        (address asset, bool erc20Wrapper) = poolManager.getVaultAsset(vault);
```

Comment:
I'm a strong proponent of boolean variables that are called `is*` . So in this case I propose `isWrapper`

Reply:
I'm fine with that too, will change